CHANGELOG.asciidoc

```diff
@@ -102,6 +101,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 - Add `key` metricset to the Redis module. {issue}9582[9582] {pull}9657[9657] {pull}9746[9746]
 - Add `socket_summary` metricset to system defaults, removing experimental tag and supporting Windows {pull}9709[9709]
+- Add `nats` module with `stats` metricset. {pull}9825[9825]
```

Comment:
Nice find :-) Could you perhaps move both of these lines to `CHANGELOG.next.asciidoc`. We recently introduced this.

Reply:
Sure! I'm in for addressing this in follow up PRs, in order to have it merged and add new metricsets on top of schema changes introduced here.

cc: @skatsaounis, @MichaelKatsoulis

Reply:
Hi both @ruflin, @ChrsMark. I will apply this change to the second commit I am preparing for the new pr regarding routes metricset #10149 

cc: @MichaelKatsoulis